public/app/features/plugins/admin/components/PluginInsights.tsx

```diff
@@ -0,0 +1,149 @@
+import { css } from '@emotion/css';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { Trans } from '@grafana/i18n';
+import { Stack, Text, TextLink, CollapsableSection, Tooltip, Icon, ColorPicker, useStyles2 } from '@grafana/ui';
+
+import { CatalogPluginInsights, ScoreLevel, InsightLevel } from '../types';
+
+type Props = { pluginInsights: CatalogPluginInsights | undefined };
+
+const getColor = (level: ScoreLevel) => {
```

Comment:
you should be using our theme words instead of hardcoded hex values https://developers.grafana.com/ui/latest/index.html?path=/docs/foundations-theme--docs